core/templates/domain/statistics/exploration-stats.model.ts

```diff
@@ -124,8 +125,9 @@ export class ExplorationStats {
   createNewWithStateRenamed(
       oldStateName: string, newStateName: string): ExplorationStats {
     const newStateStatsMapping = new Map(this.stateStatsMapping);
+    const stateStats = this.getStateStats(oldStateName);
     newStateStatsMapping.set(
-      newStateName, this.stateStatsMapping.get(oldStateName));
+      newStateName, stateStats);
```

Comment:
This doesn't need to be on a separate line.

Reply:
done